### PR TITLE
Closes #3657 Update some documentation links

### LINKED ISSUE #3657: Update some documentation links in the plugin
We need to update a few of the articles that are featured in the Beacon (https://jumpshare.com/v/zioongh75XrnIVZFD2Vj) , and also a few of the "Need Help" buttons. 

All the requested changes are here: https://docs.google.com/spreadsheets/d/1EM10uUWFrZBS-MWq-j7Ut4iua5wAkHzWy_CTahm_ZHQ/edit?usp=sharing
Note there are 2 tabs on the sheet - 1 for beacon articles, 1 for "need help" buttons
I hope the sheet is clear 😅 
Feel free to ping me if clarification is needed!!

### PR DESCRIPTION
**Pull request by engahmeds3ed**
## Description

Update documentation links on the WP Rocket settings page

Fixes #3657

## Type of change

- [x] Enhancement (non-breaking change which adds functionality)

## Is the solution different from the one proposed during the grooming?

No

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Manually

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by remyperona** [comment]
This is blocking 3.8.8 final QA and release, what's the status on this PR?

File: inc/Engine/Admin/Beacon/Beacon.php [hunk 1]
```diff
@@ -422,8 +424,8 @@ public function get_suggest( $doc_id ) {
 				],
 			],
 			'dns_prefetch'               => [
-				'en' => '541780fde4b005ed2d11784c',
-				'fr' => '5693d582c69791436155d645',
+				'en' => '5e055a602c7d3a7e9ae5881c',
+				'fr' => '5e1891892c7d3a7e9ae60983',
```
Comment (engahmeds3ed) [inline inc/Engine/Admin/Beacon/Beacon.php]: those removals are not in the sheet

File: inc/Engine/Admin/Beacon/Beacon.php [hunk 2]
```diff
@@ -489,33 +495,35 @@ public function get_suggest( $doc_id ) {
...
 			],
-			'always_purge'               => [
+			'exclude_user_agent'          => [
 				'en' => [
-					'id'  => '555c7e9ee4b027e1978e17a5',
-					'url' => 'https://docs.wp-rocket.me/article/78-how-often-is-the-cache-updated/?utm_source=wp_plugin&utm_medium=wp_rocket',
-				],
-				'fr' => [
-					'id'  => '568f7df49033603f7da2ec72',
-					'url' => 'https://fr.docs.wp-rocket.me/article/171-intervalle-cache/?utm_source=wp_plugin&utm_medium=wp_rocket',
```
Comment (engahmeds3ed) [inline inc/Engine/Admin/Beacon/Beacon.php:499]: those also not mentioned on the removed articles on the sheet

File: inc/Engine/Admin/Beacon/Beacon.php [hunk 3]
```diff
@@ -489,33 +495,35 @@ public function get_suggest( $doc_id ) {
...
-					'url' => 'https://docs.wp-rocket.me/article/121-wp-admin-area-is-slow/?utm_source=wp_plugin&utm_medium=wp_rocket',
+					'id'  => '60259156b3ebfb109b58182d',
+					'url' => 'https://docs.wp-rocket.me/article/1443-database-optimizations-are-not-working/?utm_source=wp_plugin&utm_medium=wp_rocket',
 				],
 				'fr' => [
-					'id'  => '5697cebbc69791436155ed5e',
-					'url' => 'https://fr.docs.wp-rocket.me/article/260-la-zone-d-administration-wp-est-lente/?utm_source=wp_plugin&utm_medium=wp_rocket',
+					'id'  => '60259156b3ebfb109b58182d',
+					'url' => 'https://fr.docs.wp-rocket.me/article/1486-les-optimisations-de-la-base-de-donnees-ne-fonctionne-pas/?utm_source=wp_plugin&utm_medium=wp_rocket',
 				],
```
Comment (engahmeds3ed) [inline inc/Engine/Admin/Beacon/Beacon.php:516]: also not mentioned on the sheet

File: inc/Engine/Admin/Beacon/Beacon.php [hunk 4]
```diff
@@ -533,8 +541,8 @@ public function get_suggest( $doc_id ) {
 					'url' => 'https://docs.wp-rocket.me/article/1307-rocketcdn/?utm_source=wp_plugin&utm_medium=wp_rocket',
 				],
 				'fr' => [
-					'id'  => '5e5e36712c7d3a7e9ae89555',
-					'url' => 'https://fr.docs.wp-rocket.me/article/1308-rocketcdn/?utm_source=wp_plugin&utm_medium=wp_rocket',
```
Comment (engahmeds3ed) [inline inc/Engine/Admin/Beacon/Beacon.php:537]: also this not mentioned on the sheet

**Review by remyperona — APPROVED** [review]

**Comment by remyperona** [comment]
A couple of notes for the QA:
- For the tools & tutorials sections, the list of articles shown in Beacon are populated from a configuration on HelpScout side, not ours
- I have added the articles listed in the Gdoc for each of those, but it's always showing 5 articles, and it's not something that we can change
- There might be more changes to do on the HelpScout side, notably when opening Beacon directly on a specific tab

**Review by Mai-Saad — COMMENTED** [review]
Please find my notes below @engahmeds3ed @Tabrisrp 

Regarding Needs help btn:
- All btns mentioned in doc open the article in beacon except for Post cleanup (seems this is because the btn is linked to multiple URLS, same occur for preload cache "didn't check other links not mentioned in doc" ), need to confirm that this is what we want.

Regarding beacon articles per tab: (only for EN version so far)
- Add part: most articles aren't displayed in the mentioned tab, however, they exist if we search for it. I know that this shall be set from helpscout as mentioned earlier, so this won't block this PR? or task of updating from helpscout will be considered to be done here?
- Remove part: most of articles to be removed from mentioned tab are still there. is this controllable from our side?
- I added color code in the beacon sheet to help clarify what needs to be adjusted
Yellow => exist while search but not displayed in mentioned tab
Green => exist in mentioned tab for add and removed from mentioned tab for remove
Red => not removed from mentioned tab for remove

General note for beacon:
while having list of articles card, right click card and open in new tab , results in opening site not found page within dashboard. ( expected here is open the article in new tab)

**Comment by remyperona** [comment]
@webtrainingwheels For the database optimization tab, when clicking on "need help?", Do we want to show a list of HS articles, or a specific article?

**Comment by remyperona** [comment]
@Mai-Saad For the articles per tab, everything can be configured from HS side and is updated live, so it doesn't block the PR. The support team will be able to do that task.

**Review by Mai-Saad — APPROVED** [review]
Works as expected then from ourside. If needs help shall open single article then this can be in another PR.

**Comment by webtrainingwheels** [comment]
@Tabrisrp the Need Help buttons go to a single article
EDIT: That seems not to be true. It seems they all open a Beacon list, but just have a specific one at the top of the list?

**Comment by Mai-Saad** [comment]
@webtrainingwheels 
> That seems not to be true. It seems they all open a Beacon list, but just have a specific one at the top of the list?

Not all need help btns behave like that i.e heartbeat, exclude files from CDN, Image dimensions ...etc open certain article

**Comment by remyperona** [comment]
I pushed a change to open the single article on need help for the DB Optimization tab

**Comment by Mai-Saad** [comment]
> I pushed a change to open the single article on need help for the DB Optimization tab

Will pull and check it across all tabs then as I think cache needs help btn too open multiple articles so we can collect all together

**Comment by Mai-Saad** [comment]
@Tabrisrp the following needs help btn open beacon with multiple articles:
1- Cache tab : all btns
2- File optimization: all btns
3- Media: Lazyload
4- Preload: Preload cache
5- CDN: CDN
Note: if certain article was open in beacon then clicking any Needs help btn of those mentioned above, currently won't change anything in beacon (as single article view is always on top of multiple articles view)

**Comment by remyperona** [comment]
@webtrainingwheels What do you think?

There is cases where it seems appropriate to show a list of articles when clicking on "need help", especially for sections with multiple options inside them.

But we can adapt this however the support team prefers.

**Comment by webtrainingwheels** [comment]
>I pushed a change to open the single article on need help for the DB Optimization tab

@Tabrisrp This is fine, thanks! If we need to change anything else we can do it in the future, it's not critical at this time.

**Pull request merged**